app/bundles/FormBundle/Translations/en_US/messages.ini

```diff
@@ -200,10 +200,14 @@ mautic.form.table.most.submitted="Most submitted forms"
 mautic.form.table.top.referrers="Top referrers"
 mautic.form.type.header="What type of form do you want to create?"
 mautic.form.type.campaign="Campaign Forms"
-mautic.form.type.campaign.description="Campaign forms can be selected to be the source of contacts when building a campaign. Some submit actions may not be available."
+mautic.form.type.one.differences="<li>Used within campaigns for contact sourcing</li>
+  <li>Limited submit actions</li>
+  <li>Focuses on contact management</li>"
+mautic.form.type.two.differences="<li>Independent forms focused on flexibility</li>
+  <li>Contains all form actions</li>
+  <li>Actions for scoring, segmentation, and integration</li>"
```

Comment:
```suggestion
  <li>Focuses on immediate actions</li>"
```
This text is too long and the second column drops down out of alignment. Here's what I'd proposed for the text in the earlier message.

Reply:
The ideal scenario is to fix the layout issue to keep both aligned 
I'm really afraid this change won't be helpful for users since "immediate actions" are too vague against "focus on contact management"

People will compare "contact management" with "immediate actions", it feels strange

Reply:
Space-between and body to body alignments are 2 different ways of solving the same issue.

"Space-between" does better when the symmetrical balance of all elements is a priority as each of them has equal space. This solution tends to generate a visually harmonic layout. Good? Of course.

Aligning the start of both bodies prioritizes alignment, a different design principle, but breaks balance, so the inferior part of the first panel looks lighter than the same part of the second panel. Does it look good? Of course, too. 

Who's wrong? Anyone, any solution will work just fine! ❤️👌